src/runtime/float.cpp

```diff
@@ -111,6 +111,22 @@ extern "C" double floordiv_float_float(double lhs, double rhs) {
     return floor(lhs / rhs);
 }
 
+Box* convertToFloat(Box* val) {
```

Comment:
I think this could be better than CPython's `CONVERT_TO_DOUBLE` implementation. Such as for exception handle in runtime source code.